### ISSUE TITLE
[Pyro Installer] -- Does not run on Windows

### ISSUE THREAD

**Issue by edster** [pyrocms/pyrocms#4141]
```
$ pyro new PyroCMS
Installing Pyro...

Warning: proc_open(): CreateProcess failed, error code - 267 in C:\Users\Sergio\AppData\Roaming\Composer\vendor\symfony\process\Process.php on line 310

  [Symfony\Component\Process\Exception\RuntimeException]
  Unable to launch a new process.

new [--tag [TAG]] [--dev] [--] [<name>]
```

Triggers due to proc_open() having issues with relative paths in Windows.

Apparently a `config.json` can be created to set paths for `home` and `cache-vcs-dir` to there actual locations to fix this.

More info here: https://github.com/composer/composer/issues/1995#issuecomment-24180597

**Comment by RyanThompson** [comment]
This kinda defaults the purpose unfortunately having to point to the project folder composer file..

**Comment by ufhy** [comment]
any one has to fix this??

**Comment by johmarjac** [comment]
I have the same issue... Why is this issue still not fixed?

**Comment by edster** [comment]
You don't need to use the installer, use `composer create-project pyrocms/pyrocms` instead.

**Comment by johmarjac** [comment]
@edster Yeah, I am trying since today morning to get this up and running, I tried that already 3 times. Once with command like you, then with Command --clean source or sth. and another time to just do something. In the end I called mydomain.com/installer and get server error 500 with IIS.

then I used the CLI installer which finally installed it. Homepage shows fine, when accessing backend it throws errors over errors. So now I am at this part trying it another time with the installer method....

I cant believe the installer has issues like proc_open does not work. That clearly lets me think of to not use pyrocms after one day of figuring out how this shit works

**Comment by edster** [comment]
I get it your frustration. But the composer command works fine. Since your on windows (as I am) need to add `--prefer-source`, so it would be `composer create-project pyrocms/pyrocms --prefer-source`

The GUI issue I know is fixed in the 3.5 branch. What errors are you getting in regards to errors all over the place when trying to log in?

Are you in our slack channel? You can get faster help with your installation issues there.
